Makefile

```diff
@@ -15,7 +15,7 @@
 GO = go
 HUGO_BUILD_FLAGS = -tags extended
 HUGO_MODULE = github.com/gohugoio/hugo
-HUGO ?= $(GO) run $(HUGO_BUILD_FLAGS) $(HUGO_MODULE)
+HUGO ?= CGO_ENABLED=1 $(GO) run $(HUGO_BUILD_FLAGS) $(HUGO_MODULE)
```

Comment:
Is this an optimization or a fix? Afaik on Linux this needs to be `0`, and it's `1` by default.